### ISSUE TITLE
TorchVision 0.3 Object Detection finetuning tutorial

### ISSUE THREAD

**Issue by Azer-ai** [pytorch/pytorch#36935]
I have been using this tutorial with a few datasets. I have found that sometimes while the AP for the bb increases the AP for the segmentation drops more than double.  I have also noticed that running the Penn-Fudan dataset the precision for the small areas is negative. Is this supposed to be ok to negative AP values❓

Also for the learning scheduler,  is it ok  for it to be placed before the evaluation on the test set?

**Comment by albanD** [comment]
Hi,

We use github issues only for bugs or feature requests.
Please use the forum to ask questions: https://discuss.pytorch.org/

If you think this is really a bug, it might be better to open that on the torchvision side: https://github.com/pytorch/vision/
